testing/libgeotiff/APKBUILD

```diff
@@ -1,24 +1,26 @@
 # Contributor: Bradley J Chambers <brad.chambers@gmail.com>
 # Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
 pkgname=libgeotiff
-pkgver=1.4.2
-pkgrel=2
+pkgver=1.5.1
+pkgrel=0
 pkgdesc="TIFF based interchange format for georeferenced raster imagery."
 url="https://trac.osgeo.org/geotiff"
-arch="all"
+arch="s390x aarch64 x86_64 ppc64le"
```

Comment:
Are they exclusive to these arches or are the not available on any 32bit arch ?

Reply:
if the latter i'd prefer to use !arch to exclude 32bit ones.

Reply:
changed to `arch="all !x86 !armhf !armv7"`